include/rocksdb/table.h

```diff
@@ -582,6 +582,17 @@ struct BlockBasedTableOptions {
   // Align data blocks on lesser of page size and block size
   bool block_align = false;
 
+  // Align data blocks on super block alignment. Avoid a data block split across
+  // super block boundaries.
+  bool super_block_align = false;
```

Comment:
Someone might set super_block_alignment_size expecting it to take effect. Is there a reason for a bool enablement flag vs. a nonsense value like 0 to disable?